babel.config.js

```diff
@@ -2,6 +2,7 @@ module.exports = function(api) {
   api.cache(true)
 
   return {
+    sourceType: 'unambiguous',
```

Comment:
Link to doc : https://babeljs.io/docs/en/options#sourcetype

> sourceType
> Type: "script" | "module" | "unambiguous"
> Default: "module"
 >   "script" - Parse the file using the ECMAScript Script grammar. No import/export statements allowed, and files are not in strict mode.
 >   "module" - Parse the file using the ECMAScript Module grammar. Files are automatically strict, and import/export statements are allowed.
    "unambiguous" - Consider the file a "module" if import/export statements are present, or else consider it a "script".


On avait quoi comme problème ?